pyintegration/tests/test_uri.py

```diff
@@ -0,0 +1,33 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+
+import unittest
+
+import jpy
+
+from deephaven2._jcompat import j_hashset
+from deephaven2.table import Table
+from deephaven2.uri import resolve
+from tests.testbase import BaseTestCase
+
+
+class UriTestCase(BaseTestCase):
+    def test_uri_local_table(self):
+        _JResolver = jpy.get_type("io.deephaven.server.uri.CsvTableResolver")
+        _JUriResolvers = jpy.get_type("io.deephaven.uri.resolver.UriResolvers")
+        _JUriResolversInstance = jpy.get_type(
+            "io.deephaven.uri.resolver.UriResolversInstance"
+        )
+        j_resolver = _JResolver()
+        j_resolver_set = j_hashset({j_resolver})
+        j_resolvers = _JUriResolvers(j_resolver_set)
+        _JUriResolversInstance.init(j_resolvers)
```

Comment:
This code, and the code above it, lead me to believe that the URI Python API is incomplete -- `resolve` does not stand on its own.  Is this bit of configuration and initialization always needed?  How do we expect users to do things?